rpc/types/src/bin.rs

```diff
@@ -134,6 +134,8 @@ define_request! {
     }
 }
 
+// TODO: add `top_block_hash` field
+// <https://github.com/monero-project/monero/blame/893916ad091a92e765ce3241b94e706ad012b62a/src/rpc/core_rpc_server_commands_defs.h#L263>
```

Comment:
### TODO
There's been a few fields that have been added to various types that need to be updated.